string label
---
float32 dist

